packages/storage-plus/src/snapshot/mod.rs

```diff
@@ -23,6 +26,11 @@ pub(crate) struct Snapshot<'a, K, T> {
 }
 
 impl<'a, K, T> Snapshot<'a, K, T> {
+    /// Example:
```

Comment:
This is only used crate internal, so these docs are for other internal devs

Reply:
I removed this whole comment.